recipes/example-new-recipe/recipe.yaml

```diff
@@ -0,0 +1,90 @@
+# This example shows how to define a recipe using the new YAML based recipe format introduced by 
+# CEP 13.
+
+# For more information about this format see: https://prefix-dev.github.io/rattler-build/latest/reference/recipe_file/
+
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Define variables in this section that you can use in other parts.
```

Comment:
I think this block should also mention the main differences across formats. Namely:

- No selectors, but if-then-else expressions (and link to docs)
- Jinja uses `${{ }}`